### ISSUE TITLE
Seurat conversion failure due to unpinned dependency

### ISSUE THREAD

**Issue by metakuni** [chanzuckerberg/single-cell-data-portal#1777]
From [Slack](https://czi-sci.slack.com/archives/C023Q1APASK/p1639775628216700):
```
Brian Raymor Dec 17th at 16:13
@kuni @ebezzi - There was a conversion failure during ingestion on dev.
https://cellxgene.dev.single-cell.czi.technology/collections/53248efe-590e-445e-80fc-0ef7b08878e5/private
This does not reproduce on staging.
```

```
ebezzi  3 days ago
...
SeuratObject\xe2\x80\x99 4.0.2 is being loaded, but >= 4.0.4 is required
...
ebezzi  3 days ago
I believe this is the root cause
```

**Comment by metakuni** [comment]
On 2021-12-16, a new version (4.0.6) of the Seurat package was released:
- https://cran.r-project.org/web/packages/Seurat/index.html

**Comment by metakuni** [comment]
As a temporary workaround, the version of the base Docker image will be pinned to the one before the date on which `Seurat` 4.0.6 was released:
* https://github.com/chanzuckerberg/single-cell-data-portal/pkgs/container/corpora-upload-base/11867810?tag=20211215

This will be deployed as a hotfix on the staging branch and deployed to prod today:
```
ebezzi  1 hour ago
to be extra safe, we will:
1. release this as a staging hotfix
2. do a prod deployment today after the hotfix has been applied (this way we're 100% sure that prod won't break)
3. merge this back to main
```

A longer term fix by pinning R package versions will be addressed by #1778.

**Comment by ebezzi** [comment]
This one has been hotfixed to all environment, closing it.
